vllm/executor/executor_base.py

```diff
@@ -50,6 +50,15 @@ def __init__(
     def _init_executor(self) -> None:
         pass
 
+    @abstractmethod
+    def _create_worker(self,
```

Comment:
The function name starting with underline should be an internal function in general practice. I'm not sure if it's a suitable abstract function name?

Reply:
many of the executors already have a _create_worker method, and it seems to be used internally, so I think this name fits?

Reply:
@jberkhahn apologies I was looking at this again and I'm actually also unsure of why we would add this in the base class, since it's only ever used in a private context. There isn't any place where we call _create_worker on a generic executor.

Reply:
declaring it here let's me force a standardized set of args to make sure all the _create_worker methods do roughly the same steps in the different executors. currently the separation of functionality is a bit different in different executors. I realize declaring a private method part of an interface is a bit unusual, there are spots where we do reach in and call private methods, so I'm going to have to declare those in the ABC at some point.

Reply:
In this case does that make sense to just promote it to `create_worker`?

Reply:
i could do that. the specific instance i'm referring to where stuff reaches in and calls private methods is stuff like the calls to _run_workers in `vllm/engine/llm_engine.py`

Reply:
ok, i went and changed it to `create_worker`. (incidentally, cpu_executor has a now private method for boostrapping it's worker, which now makes more sense with this naming scheme)

Reply:
@jberkhahn I'm sorry I still can't wrap my head around why we want this in the top level interface unless there's somewhere in the code that calls it on arbitrary executors (which could include other non-abstract methods in `ExecutorBase`).

Does this fix an existing typing issue?

It makes sense to standardize how things are done across the implementations but given the current scoping of what's the concern of each particular impl, it's not "wrong" for them to do this differently.

So concretely my suggestions would be to remove it here and keep the private `_create_worker` naming across the classes.